src/main/java/uk/ac/ebi/eva/commons/models/converters/data/MongoDBObjectToVariantEntityConverter.java

```diff
@@ -6,21 +6,23 @@
 import org.opencb.opencga.storage.mongodb.variant.DBObjectToVariantConverter;
 import org.opencb.opencga.storage.mongodb.variant.DBObjectToVariantSourceEntryConverter;
 import org.opencb.opencga.storage.mongodb.variant.DBObjectToVariantStatsConverter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.core.convert.converter.Converter;
+import uk.ac.ebi.eva.commons.models.metadata.VariantEntity;
 
-/**
- * Created by tom on 21/09/16.
- */
-public class MongoDBObjectToVariantConverter implements Converter<DBObject, Variant> {
+public class MongoDBObjectToVariantEntityConverter implements Converter<DBObject, VariantEntity> {
+
+    protected static Logger logger = LoggerFactory.getLogger(MongoDBObjectToVariantEntityConverter.class);
 
     @Override
-    public Variant convert(DBObject dbObject) {
+    public VariantEntity convert(DBObject dbObject) {
         DBObjectToVariantConverter converter =
                 new DBObjectToVariantConverter(
-                        new DBObjectToVariantSourceEntryConverter(VariantStorageManager.IncludeSrc.FULL),
+                        new DBObjectToVariantSourceEntryConverter(VariantStorageManager.IncludeSrc.NO),
```

Comment:
Just to be on the safe side, please note we use FIRST_8_FIELDS during the writing.